.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 executors:
   node-java:
```

Comment:
Less relevant, but why the naming `node-java`?

Reply:
@iuliag because we need java for the data-access IT test - the embedded dynamodb is java-based